dev-tools/scripts/refguide/htaccess.txt

```diff
@@ -1,6 +1,6 @@
 # Existing pages moved to sub path in the 9.0 guide
 RedirectMatch 301 ^/guide/(about-this-guide|relevance|solr-glossary|solr-tutorial)\.html /guide/solr/latest/getting-started/$1.html
-RedirectMatch 301 ^/guide/(aliases|audit-logging|authentication-and-authorization-plugins|basic-authentication-plugin|circuit-breakers|client-apis|cloud-screens|cluster-node-management|collection-management|collections-core-admin|configuring-logging|enabling-ssl|hadoop-authentication-plugin|indexupgrader-tool|installing-solr|jvm-settings|jwt-authentication-plugin|kerberos-authentication-plugin|mbean-request-handler|metrics-reporting|performance-statistics-reference|ping|plugins-stats-screen|replica-management|rule-based-authorization-plugin|securing-solr|security-ui|shard-management|solr-control-script-reference|solrcloud-recoveries-and-write-tolerance|solrcloud-with-legacy-configuration-files|taking-solr-to-production|thread-dump|upgrading-a-solr-cluster|zookeeper-access-control|cert-authentication-plugin)\.html /guide/solr/latest/deployment-guide/$1.html
+RedirectMatch 301 ^/guide/(aliases|audit-logging|authentication-and-authorization-plugins|basic-authentication-plugin|circuit-breakers|client-apis|cloud-screens|cluster-node-management|collection-management|collections-core-admin|configuring-logging|enabling-ssl|indexupgrader-tool|installing-solr|jvm-settings|jwt-authentication-plugin|mbean-request-handler|metrics-reporting|performance-statistics-reference|ping|plugins-stats-screen|replica-management|rule-based-authorization-plugin|securing-solr|security-ui|shard-management|solr-control-script-reference|solrcloud-recoveries-and-write-tolerance|solrcloud-with-legacy-configuration-files|taking-solr-to-production|thread-dump|upgrading-a-solr-cluster|zookeeper-access-control|cert-authentication-plugin)\.html /guide/solr/latest/deployment-guide/$1.html
```

Comment:
Instead of this, you should edit `mappings.csv` and `gen-refguide-redirects.py` similar to how the `_8_11` links for removed features in 9.x will then generate a `htaccess.txt` that redirect those pages to latest 8.x. So everything removed in 9.x should have the "latest" link redirect to a concrete 9.x version...